GovUk.Frontend.AspNetCore.Extensions/wwwroot/govuk/govuk-validation.js

```diff
@@ -32,8 +32,15 @@ function createGovUkValidator() {
     /**
      * Ensures the page title is prefixed with 'Error: ' when there is at least one .govuk-error-message displayed
      */
-    updateTitle: function () {
-      const prefix = "Error: ";
+      updateTitle: function () {
```

Comment:
You need to add/update tests for this function: https://github.com/thepensionsregulator/govuk-frontend-aspnetcore-extensions/blob/develop/GovUk.Frontend.AspNetCore.Extensions/__tests__/govuk-validation-updateTitle.tests.js